detect_secrets/filters/heuristic.py

```diff
@@ -165,6 +165,8 @@ def is_indirect_reference(line: str) -> bool:
 
         secret = request.headers['apikey']
     """
+    # Constain lines as the heuristics intention is to target relatively short function like strings
```

Comment:
`Constain` -> `Constrain`, `heuristics` -> `heuristic's` and `function like strings` -> something like `lines like function calls`

and `also` in the next line sounds weird, isn't that the primary reason for this constraint?

Reply:
also not that i have any problem at all with code comments :)  but IMO the reasoning here would have been pretty clear from git blame to anyone interested, maybe i'm just spoiled by tooling though

Reply:
Yep. That makes sense. Updated.